infra/routes/label_evaluator_geometric_cache/lambdas/cache_generator.py

```
@@ -0,0 +1,829 @@
+                    "observations": [],  # Not stored in serialized form
+                    "mean": {"dx": mean_dx, "dy": mean_dy},
+                    "std_deviation": std_deviation,
+                })
+        else:
+            # Legacy format: dict with pair_key -> geometry
+            for pair_key, geometry in label_pair_geometry.items():
+                if isinstance(pair_key, str):
+                    try:
+                        pair_key = eval(pair_key)  # noqa: S307
+                    except Exception:
+                        continue
```

Ruff (0.14.10)

`try`-`except`-`continue` detected, consider logging the exception

(S112)